sections/legacy-event-types.txt

```diff
@@ -517,6 +518,118 @@ completeness.
 		++---+-------------+------------------------+---------------------+--------------------------------+
 		</div>
 
+<h3 id="legacy-textevent-events">Legacy {{TextEvent}} events</h3>
+
+<h4 id="interface-textevent">Interface TextEvent</h4>
+
+
+	<pre class="idl">
+	[Exposed=Window]
+	interface TextEvent : Event {
```

Comment:
`TextEvent` inherits `UIEvent` in Chromium
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/events/text_event.h;l=38;drc=052831f0220b79fe0c3343b49f6d2863ea6de05d
and WebKit:
https://searchfox.org/wubkat/rev/77fb3649ec46dea08154a9f96e0d06fd5b9cb0bf/Source/WebCore/dom/TextEvent.idl#28

Reply:
FYI: `document.createEvent("TextEvent") instanceof UIEvent` returns `true` on Chrome.

Reply:
Yep, I had noticed and fixed this already. :)
https://github.com/w3c/uievents/blob/f36ff37adac5bbbfa7bcd8b1b097dad5479cef2f/event-algo.bs#L958